software/database/pom.xml

```diff
@@ -107,18 +107,10 @@
             <artifactId>groovy-all</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.beust</groupId>
-            <artifactId>jcommander</artifactId>
-        </dependency>
-        <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.google.code.findbugs</groupId>
-            <artifactId>jsr305</artifactId>
```

Comment:
Why remove this? It's used for `@Nullable`, which is used in `DatastoreMixins` etc. It's also a transitive dependency from brooklyn-api etc.

I believe the convention that @rdowner implemented was that if a package directly uses a class then it includes the dependency as an explicit dependency.

I can't remember whether that is still the convention - I vaguely remember a discussion and some disagreement about it previously. Personally, I have no strong feelings except that we are consistent.

@rdowner can you comment please?